projects/jdwp/defs/schema.py

```diff
@@ -16,6 +16,8 @@ class PrimitiveType(Enum):
 
     STRING = "string"
     BOOLEAN = "boolean"
+    INT = "int"
```

Comment:
Check lines 39 and 40. INT and BYTE were moved to a separate enum for stricter typing of array lengths. We can skip NewTypes for them, we can assume that IntegralType should always resolve to int in generated code.